spec/factories/embargos.rb

```diff
@@ -14,7 +14,7 @@
 FactoryGirl.define do
   factory :embargo do
     info_request
-    publish_at Time.zone.today + 3.months
+    publish_at (Time.zone.now + 3.months).beginning_of_day
```

Comment:
For continuity with the model, should this be beginning of day then + 3 months, rather than + 3 months then beginning of day? (I can't put my finger on an example when this would matter, but it feels like there'd be a leap second/year loophole somewhere).